src/common.jl

```diff
@@ -316,7 +316,15 @@ julia_color(color)::JuliaColorType = julia_color(crayon_256_color(color))
     nothing
 end
 
-out_stream_size(out_stream::Union{Nothing,IO}) = out_stream === nothing ? (40, 15) : displaysize(out_stream)
+# standard terminals seem to respect a 4:3 aspect ratio
+# unix.stackexchange.com/questions/148569/standard-terminal-font-aspect-ratio
+const ASPECT_RATIO = Ref(4 / 3)
+
+# default display size for the default BrailleCanvas (which has aspect ratio = 2) ==> (40, 15)
+const DEFAULT_HEIGHT = Ref(15)
+const DEFAULT_WIDTH = Ref(2round(Int, ASPECT_RATIO[] * DEFAULT_HEIGHT[]))
```

Comment:
Do we want to allow modifying these defaults on runtime? If not then we can use plain constants like `const ASPECT_RATIO = 4/3`.

Reply:
Another reference I've found about this: https://retrocomputing.stackexchange.com/questions/5629/why-did-80x25-become-the-text-monitor-standard.

There is no modern reason for `4:3` aspect ratio, it's just historic (Fortran, punch cards, ...). This ratio was hidden before in the (40, 15) default size in `plot.jl`, I think it's better to make it explicit so that we understand how (40, 15) was computed.